tools/pkg/platforms/centos7/files/build/mongooseim.spec

```
@@ -1,31 +1,45 @@
-%undefine _missing_build_ids_terminate_build
-%define _libdir /usr/lib/mongooseim
-%define _etcdir /etc
-%define _bindir /usr/bin
-%define _vardir /var
-
-Summary: %{_summary}
-Name: %{_name}
-Version: %{_vers}
-Release: %{_release}
-Source: %{name}_%{version}.tar.gz
-License: %{_license}
-Group: "Development/Languages"
-BuildArch: @ARCH@
-BuildRoot: %{_topdir}/BUILDROOT/%{name}-%{version}.@ARCH@
-BuildRequires:  esl-erlang >= 21.3, esl-erlang <= 22.3
-BuildRequires:  expat-devel
-BuildRequires:	chrpath
-Requires(pre): shadow-utils
-
-Vendor: %{_vendor}
+Name:              mongooseim
+Version:           %{version}
+Release:           %{release}
+Summary:           A mobile messaging platform with focus on performance and scalability
+
+Group:             Development/Languages
+License:           GPLv2
+URL:               http://packages.erlang-solutions.com
+Packager:          Erlang Solutions Ltd.
+Source0:           mongooseim.service
+
+
+BuildArch:         %{architecture}
+
+BuildRequires:     esl-erlang >= %{esl_erlang_ver_down}, esl-erlang <= %{esl_erlang_ver_up}
 
 %description
 MongooseIM is Erlang Solutions' robust and efficient XMPP server aimed at large
 installations. Specifically designed for enterprise purposes, it is
 fault-tolerant, can utilize resources of multiple clustered machines and easily
 scale in need of more capacity (by just adding a box/VM).
 
+%prep
+%setup -T -D
+cp %{SOURCE0} .
+
+%install
+./tools/configure with-all user=root prefix=/ system=yes
+sed -i 's#PREFIX=\"/\"#PREFIX=\"%{buildroot}\"#' configure.out
+make install
+sed -i 's#RUNNER_USER=\"root\"#RUNNER_USER=\"mongooseim\"#' %{buildroot}/usr/bin/mongooseimctl
+sed -i 's#RUNNER_USER=\"root\"#RUNNER_USER=\"mongooseim\"#' %{buildroot}/usr/lib/mongooseim/bin/mongooseimctl
+sed -i 's#RUNNER_USER=root#RUNNER_USER=\"mongooseim\"#' %{buildroot}/usr/lib/mongooseim/bin/mongooseim
+
+# Removed due to not specified python version in shbang
+# https://fedoraproject.org/wiki/Changes/Make_ambiguous_python_shebangs_error
+rm %{buildroot}/usr/lib/mongooseim/lib/re2-1.7.5/c_src/re2/re2/make_unicode_casefold.py
```

Is it possible to use a path with a wildcard like below?
```suggestion
rm %{buildroot}/usr/lib/mongooseim/lib/re2-*/c_src/re2/re2/make_unicode_casefold.py
```

I'm afraid the path will stop working when the `re2` lib is upgraded.